.travis.yml

```diff
@@ -1,6 +1,22 @@
 language: python
-python:
-- '3.7'
+jobs:
+  include:
+    - name: Python 3.7 on Linux
```

Comment:
The way we build the "pyinstaller" means the user shouldn't need to worry which version of python. So we should probably pick one and stick with it. I'm all for moving to python 3.8 instead of worrying which things are in 3.8 only etc.